daisy/workflow/common.go

```diff
@@ -133,6 +131,12 @@ func substitute(s reflect.Value, replacer *strings.Replacer) {
 
 		raw := f.Interface()
 		switch raw.(type) {
+		case *Workflow:
+			switch s.Interface().(type) {
+			case SubWorkflow:
```

Comment:
That is incorrect. For one, e doesn't even exist at this point. Also, to be more versatile to changes to SubWorkflow, the default case should look more like the default case below. After futzing around with this for a good 15 minutes, I find it is best to revert this change back to my previous commit.